scripts/local_prometheus/prometheus.yml

```diff
@@ -0,0 +1,7 @@
+global:
+  scrape_interval: 5s
+scrape_configs:
+  - job_name: 'local_scrape'
+    scrape_interval: 1s
+    static_configs:
+      - targets: ['0.0.0.0:6711', '0.0.0.0:6722', '0.0.0.0:6733', '0.0.0.0:6744', '0.0.0.0:6755', '0.0.0.0:9100']
```

Comment:
Should be handled within docker-compose and using service discovery, not hardcoded ports.